packages/client/src/validation.ts

```diff
@@ -168,8 +168,9 @@ export function validateValueFromSchema(
 	}
 
 	try {
-		const validate = ajv.compile( args );
-		const valid = validate( value );
+		const { default: defaultValue, ...schemaWithoutDefault } = args;
+		const validate = ajv.compile( schemaWithoutDefault );
+		const valid = validate( value === undefined ? defaultValue || {} : value );
```

Comment:
These schemas can have simple types like `string`, so I'm curious what was the decision process for using `{}`. Should it keep it undefined when the default value is not found? I think it's what happens on the server:

https://github.com/WordPress/wordpress-develop/blob/2740a8ed55af56ccf8c5a548dd393bdd3e8472c8/src/wp-includes/abilities-api/class-wp-ability.php#L448-L450